anyblok/bloks/anyblok_core/system/blok.py

```diff
@@ -280,10 +280,13 @@ def load(self):
     @classmethod
     def load_all(cls):
         """Load all the installed bloks"""
-        query = cls.query().filter(cls.state == "installed")
-        bloks = query.order_by(cls.order).all()
-        if bloks:
-            bloks.load()
+        query = cls.select_sql_statement()
+        query = query.where(cls.state == "installed")
+        query = query.order_by(cls.state)
```

Comment:
why moving from `cls.order` to `cls.state`, I'm wondering if that impact with blok dependencies.

It can be a good idea to add index on filed used in SQL ORDER BY instructions if not already placed if you are trying to find perf issue